run/rails-cat_album/config/credentials.yml.enc

```diff
@@ -1 +0,0 @@
-CLAdHohL7X/1S9okyy7v40KQeH0YqmXKcpZQ8gLWA1+vpZbQ0Qvw6OzvNYCqF2LKD5DdQPT9AoiAVTRyv7Etc0rG3QL1MUFsz5oWUnWxhQthXB34TnXUPGMgX7Z5ljm/vJRAbDHcnGEHDtX9PCkMmBe/qPvm64BONJuUpEKVVD3u5esGZBPEphTnDWD1c7+VuzSry9uE0Np/EVUbGtsYbvwDNr76ELV4YoLA7/y3J1pxuUIy8BbfGAHrdWZA3Bdts3deG+4DL3X9ULoBUeS2z7ye7KCcbqBY0QkJiiMo4XgZ71FjbQeXV4HSaBaGAyYXDvgPt0MTyQR6Zor4++T9n1S3GdYyA2h6lvvDTgFj75q9YPtd0nQ5AuV0vYw5eqB8C7yANXbkRN2SCn25S7ouh2xZSW3vKz6WwPsd--77zv09OfoZmDXKFo--KVsB0cDA5AA7ugy6j3VGcg==
```

Comment:
Is this file meant to be deleted? I was under the impression this is how RAILS_MASTER_KEY works: It provides the key to decrypt this file, and this file then includes the db password, etc.

Reply:
Yes, I meant to delete the file because when people clone this repo, they'll have to generate their own credentials and master.key files to store their rails credentials(db password). I didn't want to have my credentials file uploaded, only to tell them they would need to delete it and regenerate both files on their own again.

Reply:
oh right, good point.